### PR TITLE
Calling SPM configuration helper from static ctor

### PR DESCRIPTION
**Pull request by abatishchev**

### CODE REVIEW

File: Easy.Common/RestClient.cs [hunk 1]
```diff
@@ -351,6 +350,18 @@ public void Dispose()
             _client.Dispose();
             lock (_endpoints) { _endpoints.Clear(); }
         }
+        
+        /// <summary>
+        /// Makes the necessary changes to <see cref="ServicePointManager" /> properties.
+        /// </summay>
+        public static void ConfigureServicePointManager()
```
Comment (abatishchev) [inline Easy.Common/RestClient.cs]: I've made it public just in case user would want to re-apply these changes by some reason

Comment (NimaAra) [reply Easy.Common/RestClient.cs]: I prefer it to remain `private`. If anyone wants to override they can do so outside the context of the `RestClient` or wrap it inside their own abstraction.

Comment (abatishchev) [reply Easy.Common/RestClient.cs]: Fair enough, reverted

**Comment by NimaAra** [comment]
Excellent, thank you.

**Pull request merged**